inference-engine/tests/functional/inference_engine/ir_serialization/models/add_abc_initializers_nan_const.prototxt

```diff
@@ -0,0 +1,95 @@
+ir_version: 3
```

Comment:
Looks like you missie adding this .prototxt to INSTANTIATE_TEST_CASE_P(ONNXSerialization)